modules/core/src/main/java/org/apache/ignite/internal/processors/platform/client/ClientStatus.java

```diff
@@ -37,6 +37,12 @@ private ClientStatus (){
     /** Invalid op code. */
     public static final int INVALID_OP_CODE = 2;
 
+    /** Functionality is disabled. */
+    public static final int FUNCTIONALITY_DISABLED = 100;
+
+    /** Resource limit exceed. */
+    public static final int RESOURCE_LIMIT_EXCEED = 101;
```

Comment:
Let's not generalize too much, make this only about compute, and name it like the one above: `TOO_MANY_COMPUTE_TASKS`